contracts/NotifierManager.sol

```diff
@@ -39,6 +39,7 @@ contract NotifierManager is OwnableUpgradeable, PausableUpgradeable {
         address consumer;
         bytes providerSignature;
         uint256 balance;
+        string apiKey;
```

Comment:
The api key cannot be exposed to public in a smart contract. Then anyone can use that key.